.github/workflows/backend-api-push-to-main.yml

```
@@ -144,3 +148,17 @@ jobs:
           signing-profile-name: ${{ secrets.BACKEND_API_DEV_SIGNING_PROFILE }}
           template-file: .aws-sam/build/template.yaml
           working-directory: backend-api
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@55f725fcb586ca16a5ed5b0d75d464defbfa831b #main
+        with:
+          aws-region: eu-west-2
+          role-to-assume: ${{ secrets.BACKEND_API_BUILD_GH_ACTIONS_ROLE_ARN }}
+
+      - name: Upload SAM artifact into the BUILD artifact bucket
+        uses: govuk-one-login/devplatform-upload-action@6985ccbaa306e2320a8826252c922af65242b283 #main
+        with:
+          artifact-bucket-name: ${{ secrets.BACKEND_API_BUILD_ARTIFACT_BUCKET }}
+          signing-profile-name: ${{ secrets.BACKEND_API_BUILD_SIGNING_PROFILE }}
```

```suggestion
          signing-profile-name: ${{ secrets.BUILD_SIGNING_PROFILE_NAME }}
```

With the secret amendment